xml/System/Enum.xml

```diff
@@ -4132,7 +4132,7 @@ thisInstance And flag = flag
       <Docs>
         <typeparam name="TEnum">The enumeration type to which to convert <paramref name="value" />.</typeparam>
         <param name="value">The case-sensitive string representation of the enumeration name or underlying value to convert.</param>
-        <param name="result">When this method returns, contains an object of type <c>TEnum</c> whose value is represented by <paramref name="value" /> if the parse operation succeeds. If the parse operation fails, contains the default value of the underlying type of <c>TEnum</c>. This value need not be a member of the <c>TEnum</c> enumeration. This parameter is passed uninitialized.</param>
+        <param name="result">When this method returns, contains an object of type <c>TEnum</c> whose value is represented by <paramref name="value" /> if the parse operation succeeds. If the parse operation fails, contains the default value of the underlying type of <c>TEnum</c>. The default value is the first element on the <c>TEnum</c> or a member in the enum with a value assigned of 0. This value need not be a member of the <c>TEnum</c> enumeration. This parameter is passed uninitialized.</param>
```

Comment:
```suggestion
        <param name="result">When this method returns, contains an object of type <c>TEnum</c> whose value is represented by <paramref name="value" /> if the parse operation succeeds. If the parse operation fails, contains the default value of the underlying type of <c>TEnum</c>. This parameter is passed uninitialized.</param>
```

Reply:
I'd suggest adding this info into the Remarks section below instead. Thoughts?

> If the parse operation fails, the `result` parameter contains the default value of the underlying type of `TEnum`. The default value is the first element on the `TEnum` or a member in the enumeration with an assigned value of 0. This value need not be a member of the `TEnum` enumeration.

But I also have a question - if the default value is the first element or the member with a value of 0, why does it say "this value need not be a member of the enum"?

Reply:
>  if the default value is the first element or the member with a value of 0, why does it say "this value need not be a member of the enum"?

The default value is 0 (numerically), which is by default equal to the first element of the Enum, if the dev assigned custom numbers then default Enum not have to be the 1st element or can be even not exist within the enum members. Example:
```cs
public enum DefaultValueEnum
{
    First, // defaulted to 0
    Second, // 1
    Third // 2
}

// The case where default 0 not exist:
public enum EnumWithCustomValue
{
    First = 1, 
    Second = 2, 
    Third  = 3
}
```
So not sure if we really need this update here, or the wording needs to be improved

Reply:
From the sample in the SO link provided an assignment to zero is used if the dev doesn't want to use/modify the current order of the enum members. If there is no zero assigned on any member, but all members have a value return then tryparse returns a 0.
``` C#
        enum SubscriptionStatusEnumExtension
        {
            Unrecognized = 3,
            Check = 2,
            OK = 1,
        }
```
This return the value 0 as default and not any of the enum members.

Reply:
``` c#
        enum SubscriptionStatusEnumExtension
        {
            UnRecognized = 3,
            CHECK = 2,
            OK = 0,  // OK will be the default
        }
```